### PR TITLE
Update documentation for `draw` control wires change

### PR DESCRIPTION
**Pull request by albi3ro**
In #2668 , the draw transform changed to use `●` and `○` to denote control wires.  This PR updates all the documentation to be compatible with this change.

### CODE REVIEW

File: pennylane/transforms/qcut.py [hunk 1]
```diff
@@ -428,9 +428,10 @@ def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
     >>> graph = qml.transforms.qcut.tape_to_graph(tape)
     >>> tape = qml.transforms.qcut.graph_to_tape(graph)
     >>> print(tape.draw())
-     0: ──RX(0.4)──────╭C───────────────╭X──┤ ⟨Z⟩
-     1: ──RY(0.5)──────╰X──MeasureNode──│───┤
-     2: ──PrepareNode───────────────────╰C──┤
+    0: ──RX──────────╭●──────────────╭X─┤  <Z>
+    1: ──RY──────────╰X──MeasureNode─│──┤
```
Comment (AmintorDusko) [inline pennylane/transforms/qcut.py:432]: I'm curious, why are the phases omitted in such a case?

Comment (albi3ro) [reply pennylane/transforms/qcut.py:432]: This documentation must have been written prior to the the new `qml.drawer.tape_text` method being called by `tape.draw`.  `qml.drawer.tape_text`, and thus `tape.draw`, omit parameters by default.  They must be requested by `decimals=2`.

Comment (AmintorDusko) [reply pennylane/transforms/qcut.py:432]: Nice! Thank you for the clarification.

**Comment by AmintorDusko** [comment]
Hi @albi3ro, Looks good, I'm ready to approve. Could you please re-trigger the tests? 
Something happened with it.

**Review by AmintorDusko — APPROVED** [review]
Nice work!

**Pull request merged**